fortitude/src/rules/mod.rs

```diff
@@ -74,92 +70,89 @@ pub fn code_to_rule(category: Category, code: &str) -> Option<(RuleGroup, Rule)>
+        (Correctness, "091") => (RuleGroup::Stable, Ast, Default, correctness::external::ExternalProcedure),
+        (Correctness, "101") => (RuleGroup::Preview, Ast, Default, correctness::derived_default_init::MissingDefaultPointerInitalisation),
+        (Correctness, "111") => (RuleGroup::Stable, Ast, Default, correctness::external_functions::ProcedureNotInModule),
+        (Correctness, "121") => (RuleGroup::Stable, Ast, Default, correctness::use_statements::UseAll),
+        (Correctness, "122") => (RuleGroup::Preview, Ast, Default, correctness::use_statements::MissingIntrinsic),
+        (Correctness, "131") => (RuleGroup::Preview, Ast, Default, correctness::accessibility_statements::MissingAccessibilityStatement),
+        (Correctness, "132") => (RuleGroup::Preview, Ast, Optional, correctness::accessibility_statements::DefaultPublicAccessibility),
+        (Correctness, "141") => (RuleGroup::Preview, Ast, Optional, correctness::include_statement::IncludeStatement),
+        
+        (Modernisation, "001") => (RuleGroup::Stable, Ast, Optional, modernisation::double_precision::DoublePrecision),
+        
+        (Portability, "001") => (RuleGroup::Preview, Ast, Optional, portability::magic_io_unit::NonPortableIoUnit),
+        (Portability, "011") => (RuleGroup::Stable, Ast, Default, portability::literal_kinds::LiteralKind),
+        (Portability, "012") => (RuleGroup::Stable, Ast, Default, portability::literal_kinds::LiteralKindSuffix),
+        (Portability, "021") => (RuleGroup::Stable, Ast, Default, portability::star_kinds::StarKind),
+
+        (Style, "001") => (RuleGroup::Stable, Text, Default, style::line_length::LineTooLong),
+        (Style, "021") => (RuleGroup::Stable, Ast, Default, style::exit_labels::MissingExitOrCycleLabel),
+        (Style, "041") => (RuleGroup::Stable, Ast, Default, style::old_style_array_literal::OldStyleArrayLiteral),
+        (Style, "051") => (RuleGroup::Stable, Ast, Default, style::relational_operators::DeprecatedRelationalOperator),
+        (Style, "061") => (RuleGroup::Stable, Ast, Default, style::end_statements::UnnamedEndStatement),
+        (Style, "071") => (RuleGroup::Stable, Ast, Default, style::double_colon_in_decl::MissingDoubleColon),
+        (Style, "081") => (RuleGroup::Preview, Ast, Default, style::semicolons::SuperfluousSemicolon),
+        (Style, "082") => (RuleGroup::Preview, Ast, Optional, style::semicolons::MultipleStatementsPerLine),
+        (Style, "091") => (RuleGroup::Stable, Path, Default, style::file_extensions::NonStandardFileExtension),
+        // There are likely to be many whitespace rules at some point, reserve S1xx for them
+        (Style, "101") => (RuleGroup::Stable, Text, Default, style::whitespace::TrailingWhitespace),
+        (Style, "102") => (RuleGroup::Stable, Ast, Optional, style::whitespace::IncorrectSpaceBeforeComment),
+        (Style, "201") => (RuleGroup::Stable, Ast, Default, style::implicit_none::SuperfluousImplicitNone),
```

Comment:
I don't know if this is as useful as a default rule. The missing implicit none rule is definitely one to be on by default, but a superfluous statement doesn't seem like it is bad enough to justify being on by default.

Reply:
It is a `style` rule though, and as long as we're upfront that these are opinionated, that's maybe the justification? I'm mildly in favour of keeping it on my default, but not opposed to off by default.

Reply:
I wouldn't really mind either way.

Reply:
I had a think about it and decided to set it to optional. I don't see it as a particularly severe style violation, and I know some style guides recommend putting it everywhere (I think it's to make sure you're protected in case you ever decide to move a function into a module that deliberately isn't using `implicit none` for whatever reason).